amnestywebsite/humanity-theme
# Adds the 50/50 text with video block patterns

Ref: https://github.com/amnestywebsite/humanity-theme/issues/177

Adds the 50/50 text with video block patterns

**Steps to test**:
1. Edit a post
2. Open the block inserter and check the new pattern appears under the patterns tab
3. Insert the pattern and check it matches the example in the ticket

Example: http://bigbite.im/i/g5xjAA
